R/vdiagram-render.R

```diff
@@ -54,6 +54,129 @@ renderVDiagram <- function(data){
 }
 
 createVdiagram <- function(data) {
-  data <- parseVDiagramData(data)
-  vplot(data)
+  styles <- getVDiagramStyle()
+  
+  vdiagramData <- parseVDiagramData(data)
+  
+  vplot <- gsplot() %>%
+    points(NA,NA, ylab=styles$labels$y, xlab=styles$labels$x) %>%
+    callouts(y=c(vdiagramData$minStage, vdiagramData$maxStage), styles$callouts) %>%
```

Comment:
does `styles$callouts` unpack multiple arguments? Otherwise this is getting assigned to `x`, right? (since y is named and x is the first argument)